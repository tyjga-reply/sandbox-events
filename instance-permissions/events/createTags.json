{
    "version": "0",
    "id": "b5dcc4cc-f10c-7250-0f4f-cf00a54fda21",
    "detail-type": "AWS API Call via CloudTrail",
    "source": "aws.ec2",
    "account": "211125361846",
    "time": "2024-12-15T15:30:30Z",
    "region": "us-east-1",
    "resources": [],
    "detail": {
      "eventVersion": "1.10",
      "userIdentity": {
        "type": "AssumedRole",
        "principalId": "AROATCKANLS3NIDXYUGTV:t.kapp@reply.de",
        "arn": "arn:aws:sts::211125361846:assumed-role/AWSReservedSSO_AdministratorAccess_d519e2a9ed8243a4/t.kapp@reply.de",
        "accountId": "211125361846",
        "accessKeyId": "ASIATCKANLS3FCTHHCSH",
        "sessionContext": {
          "sessionIssuer": {
            "type": "Role",
            "principalId": "AROATCKANLS3NIDXYUGTV",
            "arn": "arn:aws:iam::211125361846:role/aws-reserved/sso.amazonaws.com/eu-central-1/AWSReservedSSO_AdministratorAccess_d519e2a9ed8243a4",
            "accountId": "211125361846",
            "userName": "AWSReservedSSO_AdministratorAccess_d519e2a9ed8243a4"
          },
          "attributes": {
            "creationDate": "2024-12-15T14:23:12Z",
            "mfaAuthenticated": "false"
          }
        }
      },
      "eventTime": "2024-12-15T15:30:30Z",
      "eventSource": "ec2.amazonaws.com",
      "eventName": "CreateTags",
      "awsRegion": "us-east-1",
      "sourceIPAddress": "92.72.32.48",
      "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/18.1.1 Safari/605.1.15",
      "requestParameters": {
        "resourcesSet": {
          "items": [
            {
              "resourceId": "i-0edd63d2aaa65dfb2"
            }
          ]
        },
        "tagSet": {
          "items": [
            {
              "key": "test",
              "value": "random"
            }
          ]
        }
      },
      "responseElements": {
        "requestId": "7df677c7-ad37-448a-b4c4-cb5e205d33c6",
        "_return": true
      },
      "requestID": "7df677c7-ad37-448a-b4c4-cb5e205d33c6",
      "eventID": "19cc904d-537e-4f33-a3e6-2554d3a2c565",
      "readOnly": false,
      "eventType": "AwsApiCall",
      "managementEvent": true,
      "recipientAccountId": "211125361846",
      "eventCategory": "Management",
      "tlsDetails": {
        "tlsVersion": "TLSv1.3",
        "cipherSuite": "TLS_AES_128_GCM_SHA256",
        "clientProvidedHostHeader": "ec2.us-east-1.amazonaws.com"
      },
      "sessionCredentialFromConsole": "true"
    }
  }