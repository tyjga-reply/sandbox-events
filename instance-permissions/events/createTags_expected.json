{
    "statusCode": 200,
    "body": "{\"message\": \"Resource ID extracted successfully!\", \"eventName\": \"CreateTags\", \"resourceId\": \"i-0edd63d2aaa65dfb2\", \"valid\": true}"
  }